indexer/Indexer.cc

```diff
@@ -91,6 +93,14 @@ void FileLocalMacroOccurrence::emitSymbolInformation(
   // TODO: Set documentation
 }
 
+std::strong_ordering operator<=>(const NonFileBasedMacro &m1,
+                                 const NonFileBasedMacro &m2) {
+  // ASSUMPTION: built-in definitions must be in the same "header"
```

Comment:
It's not possible to assert this here, because obtaining the file information requires access to a SourceManager, but I'll try to add an assertion at the point of sorting.

Reply:
Added assert in [`f81bba4` (#104)](https://github.com/sourcegraph/scip-clang/pull/104/commits/f81bba46a78078f225801e1aa9b493e4f4de2d76)